README.md

```diff
@@ -52,9 +55,23 @@ cd django-DefectDojo
 ./dc-up.sh postgres-redis
 # obtain admin credentials. the initializer can take up to 3 minutes to run
 # use docker-compose logs -f initializer to track progress
-docker-compose logs initializer | grep "Admin password:"
+docker compose logs initializer | grep "Admin password:"
+```
+## For older verion Composer V1
```

Comment:
```suggestion
## For older verion Compose V1
```

Reply:
Hi @Maffooch,

I have made the suggested change to the code to ensure that everything is in order. I kindly request that you please take some time to re-review the pull request at your earliest convenience.

Reply:
omg how did I miss the `verion` one 🤦🏼 

Good catch @rndmh3ro